hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/healthcondition/HealthCondition.scala

```diff
@@ -21,7 +23,8 @@ sealed abstract class HealthCondition(
   val both: Option[String] = None,
   val cg: Option[String] = None,
   val dx: Option[String] = None,
-  val isOther: Boolean = false
+  val isOther: Boolean = false,
+  val descriptionSuffixOverride: Option[String] = None
```

Comment:
Good instinct, but a potential refinement: instead of `isOther` and `descriptionSuffixOverride`, could we just have a `descriptionSuffix`? It'd default to `None`, but the "other" cases would set it to "other", and this new weird case would set it to "valve"

Reply:
Yeah, I had been thinking about that. It would mean setting the descriptionSuffix to  "other_spec" for the 20 or so normal description fields, but is definitely easier to read avoids making the transformation more complex.

Reply:
@danxmoran I looked into this some more and found a couple of cases that make it difficult (or maybe impossible?) to do with a single `descriptionSuffix ` field. For example, for the "other eye" transformations (below), the description suffix would need to be `eye_other_spec` for the `dx` fields and `other_spec` for the `cg` fields.
```
  case object OtherEye
      extends HealthCondition(
        198L,
        "Other eye condition",
        Eye,
        cg = Some("other"),
        dx = Some("eye_other")
      )
```
It would be great if we could just pass in the `spec` part, but the "valve" field prevents that (the normal fields are `hs_dx_cardiac_vd_*` and the spec field is `hs_dx_cardiac_valve `.

Let me know what you think, or if it would be easier to do a quick call to talk about it.

Reply:
Cursed HLES schema... If the way you have it now is working fine, I say we just leave it.